app/presenters/event_presenter.rb

```diff
@@ -0,0 +1,29 @@
+class EventPresenter < SimpleDelegator
+  LOCAL_TIME_ZONE = "America/New_York"
```

Comment:
This may seem a little odd to hardcode the local time zone but this site is purely for the Boston Ruby Group. The Meetup API does provide the group's time zone. I considered pulling that in and using the API provided timezone but I've lost some faith in the API consistently returning information so I opted for a more safe route.